server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -285,7 +285,7 @@ protected void parseCreateField(ParseContext context) throws IOException {
 
     @Override
     public final boolean parsesArrayValue() {
```

Comment:
I think we need to remove this override and instead  have `GeoPointFieldMapper` and `PointFieldMapper` implement it and return `true`?

Reply:
woops. jumped on that too fast. you're right